src/main/resources/config.yml

```diff
@@ -90,4 +90,17 @@ motd:
     idle: gray                                           # Color designating a match which is waiting to start
     starting: yellow                                     # Color designating a match which is starting
     running: green                                       # Color designating a match which is currently running
-    finished: red                                        # Color designating a match which has ended
\ No newline at end of file
+    finished: red                                        # Color designating a match which has ended
+
+flair:
+  config-mode: false                                     # If true, use the flairs defined below,
+                                                         # otherwise use flairs sent from an API
+  definitions:
+    admin:
+      priority: 1
+      symbol: ❖
+      color: gold
+    mod:
+      priority: 2
+      symbol: ❖
+      color: red
```

Comment:
So this may not make full sense now, but with some upcoming changes it would be better to do this:

```yml
groups:
   admin:
      priority: 1
      flair:
         symbol: ❖
         color: gold
     # Future additions (not to be implemented)
     #
     # permissions:
     #   - "my.other.plugin" 
     # discord:
     #   group: ...
```